api/src/services/items.ts

```diff
@@ -69,7 +89,12 @@ export class ItemsService<Item extends AnyItem = AnyItem> implements AbstractSer
 	/**
 	 * Create a single new item.
 	 */
-	async createOne(data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
+	async createOne(data: Partial<Item>, opts: MutationOptions = {}): Promise<PrimaryKey> {
+		if (!opts.mutationTracker) opts.mutationTracker = this.createMutationTracker();
+		if (!opts.bypassLimits) {
+			opts.mutationTracker!.trackMutations(1);
```

Comment:
Should this be optional instead so the service doesnt throw fatal errors when the tracker is not set? 🤔 
```suggestion
			opts.mutationTracker?.trackMutations(1);
```

Reply:
Good eye! They were needed before and no longer needed now. Ref: https://github.com/directus/directus/pull/17535/commits/a0201608dc0317c3773b53734b195cabe8d9012c